code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -1116,10 +1116,14 @@
 	var/chance = dose*removed
 	if(M.bodytemperature < 170)
 		chance = (chance*4) + 5
+		M.adjustBrainLoss(-30 * removed)
```

Comment:
Isnt that a bit excessive ? -> 30 brain damage per unit removed

Reply:
Alkysine does the same thing.